src/class/SimpleData.ts

```diff
@@ -603,6 +604,36 @@ export default class SimpleData {
         return this
     }
 
+    @logCall()
+    addRank({
+        rankTitle,
+        rankBy,
+        sort,
+        locale,
+        handleTies,
+        overwrite = true,
+    }: {
+        rankTitle: string
+        rankBy?: string | string[]
+        sort?: true | false
+        locale?: string | (string | undefined | null | boolean)[]
+        handleTies?: "sequential" | "indexWithTie" | "indexWithoutTie"
```

Comment:
What's the difference between "sequential" and "indexWithouTie"? I feel all of these options could be simplified by using handleTies: true | false.

Reply:
They differ on whether or not the following ranks are sequential, ie follow the previous even if there were ties or if they are based on the index after the tie. There are reasons you could want both. If you like we can get rid of the sequential option and just keep the index without and without simplified to true and false